test/channels/api.js

```diff
@@ -89,7 +89,7 @@ describe('api', function() {
     });
 
     it('returns a wildcard CORS header', function(done) {
-      process.env.CORS_ENABLED = true;
+      process.env.CORS_ENABLED = 'true';
```

Comment:
This is actually unnecessary. As described at https://nodejs.org/api/process.html#process_process_env:

> Assigning a property on process.env will implicitly convert the value to a string.

I thought about it before using `true` rather than `'true'` and in the end decided that it was better to go with our mental model (a boolean) rather than what it really is (a string).

But I'm equally happy to leave it as a string...makes no difference :)